README.md

```diff
@@ -1,5 +1,33 @@
-# MSMR Project- Draft
+# Multi-System Model of Resilience (MSMR) Website
 
+---
+![About Screenshot](./docs/images/about_screenshot.png)
+
+![Resilence Screenshot](./docs/images/resilence_screenshot.png)
+
+---
+
+## About
+The Multi-System Model of Resilience (MSMR) is a copyright-protected and empirically-validated, multi-dimensional model and measure of resilience capacity across different areas in your life.
+
+The MSMR digital application provides an opportunity for users to engage and learn about their personal resilience capacity - where they stand, and how they can improve their resilience.
+
+---
+## Sponsors of Multi-System Model of Resilience Research
+<table>
+  <tr>
+    <td>Ryerson University</td>
+     <td>Royal Bank of Canada</td>
+     <td>Canadian Psychological Association</td>
+  </tr>
+  <tr>
+    <td><img src="https://msmr-blue.vercel.app/images/sponsors/Ryerson.jpg" width=200></td>
+    <td><img src="https://msmr-blue.vercel.app/images/sponsors/RBC.png" width=200 ></td>
+    <td><img src="https://msmr-blue.vercel.app/images/sponsors/CPA.jpg" width=200 ></td>
+  </tr>
+ </table>
```

Comment:
isn't there Markdown syntax to make tables too?

Reply:
@jimmyygit i am not sure, can you provide me with some links to further explore?

Reply:
Some resources from google :)
https://www.markdownguide.org/extended-syntax/#tables
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables

Reply:
@jimmyygit i am gonna just do like a nice image collage of all the sponsors in the future instead of using tables